build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -4,7 +4,10 @@
   <UsingTask TaskName="GenerateBuildVersionInfo" AssemblyFile="$(CLIBuildDll)" />
   <UsingTask TaskName="CheckPrereqs" AssemblyFile="$(CLIBuildDll)" />
   <UsingTask TaskName="SetEnvVar" AssemblyFile="$(CLIBuildDll)" />
+  <UsingTask TaskName="DotNetBuild" AssemblyFile="$(CLIBuildDll)" />
```

Comment:
It feels like these belong in a more global place that Prepare.targets.  Prepare doesn't using any of these new tasks, so it seems weird for it to be declaring them.

Maybe adding a new "global.props", or similar name, would be a good place for these UsingTask elements that are used across all our .targets files.